manubot/webpage/webpage_command.py

```diff
@@ -107,10 +107,10 @@ def checkout_existing_versions(args):
         '--',
         'v',
     ]
+    process = subprocess.run(command, stderr=subprocess.PIPE)
     logging.info(
         f"Attempting checkout with the following command:\n{shlex_join(process.args)}"
```

Comment:
Would be best if we keep the subprocess command after the logging statement but do:

```suggestion
        f"Attempting checkout with the following command:\n{shlex_join(command)}"
```